cmd/gossamer/toml_config.go

```diff
@@ -11,25 +11,36 @@ import (
 	"reflect"
 	"unicode"
 
-	"github.com/naoina/toml"
-
+	"github.com/ChainSafe/gossamer"
 	ctoml "github.com/ChainSafe/gossamer/dot/config/toml"
+	"github.com/naoina/toml"
 )
 
-// loadConfig loads the values from the toml configuration file into the provided configuration
-func loadConfig(cfg *ctoml.Config, fp string) error {
+func loadConfigFromResource(cfg *ctoml.Config, resourcePath string) error {
+	file, err := gossamer.DefaultConfigTomlFiles.Open(resourcePath)
+	if err != nil {
+		logger.Errorf("failed to open toml configuration file: %s", err)
+		return err
```

Comment:
nit maybe return a wrapped error and let the caller log this out instead (to avoid duplicate log):

```suggestion
		return fmt.Errorf("opening toml configuration file: %w", err)
```